estimator/lwe.py

```diff
@@ -42,12 +47,6 @@ def rough(cls, params, jobs=1, catch_exceptions=True):
 
 
         """
-        # NOTE: Don't import these at the top-level to avoid circular imports
```

Comment:
how did you resolve this?

Reply:
Always happy to revert the change.
I didn't observe any circular dependecies. `flake8` didn't pick up on anything (not 100% guarantee). I have a couple of thoughts on the matter, 
1. Importing in this way is contrary to PEP8. Clarity and least surprise is to aim to always import at the top of the file.
2. Circular dependency imports should not be ignored through workarounds. These workarounds ideally should be temporary measures; they are usually seen as code smells indicating a serious problem with the structure of the codebase. Instead of pre-empting, letting the problem show itself and addressing it is better in the long run for a project.

Reply:
Okay, let's see how it goes :)